libs/utils/utils.js

```diff
@@ -822,7 +822,7 @@ async function checkForPageMods() {
   }
 
   if (targetEnabled) {
-    await loadMartech({ persEnabled: true, persManifests, targetMd });
+    loadMartech({ persEnabled: true, persManifests, targetMd });
```

Comment:
Wouldn't this have the potential of causing flicker? Without `await`, page rendering continues normally, adding, say, a 'Buy Now' promo as the first element on the page. But personalisation then figures out that the user is already entitled to the product and that the promo should be removed, causing flicker and CLS. Did you set up a personalisation manifest with such a use-case and see that everything is working as expected?

Reply:
martech has alloy-hiding mechanism to fight flicker by setting the opacity to 0.01 then target is enabled.
To test against more advanced cases, I think it is better if the work is done by Target/MEP devs.
This came up as a clear impact factor on the LCP on Homepage.
I hope @chrischrischris  and @vgoodric can confirm whether this is doable or a no go.